common/changes/@microsoft/rush/master_2019-09-09-18-25.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "comment": "Fixing apply git tags during publish when using --pack and the tag already exists (existing tags not re-tagged, same as non --pack publish)",
```

Comment:
```suggestion
      "comment": "Fix an issue where Rush attempted to add Git tags for packages that had already been published when the publish command is run with the --pack flag. This caused a fatal error when tags already existed.",
```

Reply:
Updated the comment with a commit and mentioned the `--apply-git-tags-on-pack` flag. Thanks for the suggestion!